spec/components/previews/announcement_component_preview.rb

```diff
@@ -0,0 +1,18 @@
+class AnnouncementComponentPreview < ViewComponent::Preview
+  # @param message [String] text "The text to display in the announcement"
```

Comment:
A neat lookbook feature that allows your to use YARD - esque syntax to add dynamic params to the previews